pkg/controller/operator.go

```diff
@@ -172,15 +118,15 @@ func (s *Support) Run(ctx context.Context, controller *controllercmd.ControllerC
 
 	// the gatherers periodically check the state of the cluster and report any
 	// config to the recorder
-	configPeriodic := clusterconfig.New(gatherConfigClient, gatherKubeClient.CoreV1(), gatherKubeClient.CertificatesV1beta1(), metricsClient, registryClient.ImageregistryV1(), crdClient, gatherNetworkClient, dynamicClient, gatherPolicyClient, appsClient, discoveryClient)
+	clusterConfigGatherer := clusterconfig.New(gatherKubeConfig, gatherProtoKubeConfig, metricsGatherKubeConfig)
 	periodic := periodic.New(configObserver, recorder, map[string]gather.Interface{
-		"config": configPeriodic,
+		"clusterconfig": clusterConfigGatherer,
```

Comment:
This should be changed back so that we don't break the rules

Reply:
I guess I was wrong. The `config` string is really hardcoded in every gather function so probably no need to keep the original name.